numerize/numerize.py

```diff
@@ -1,69 +1,35 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-import sys
 from decimal import Decimal
 
-def round_num(n, decimals):
-    '''
-    Params:
-    n - number to round
-    decimals - number of decimal places to round to
-    Round number to 2 decimal places
-    For example:
-    10.0 -> 10
-    10.222 -> 10.22
-    '''
-    return n.to_integral() if n == n.to_integral() else round(n.normalize(), decimals)
+def round_num(n):
```

Comment:
lets keep the second param `decimals`, so the user can choose how many decimal places they want to show, but have it default to 2 if it is not provided